nextflow.config

```diff
@@ -15,6 +15,15 @@ params {
   outdir = './results'
   igenomes_base = 's3://ngi-igenomes/igenomes/'
   clusterOptions = false
+  
+  // Pipeline version
+  version = '1.4dev'
+  /*
+  * Variable to specify the docker / singularity image tag to use
+  * Picks up on use of -r 1.3 in nextflow command
+  * Stable releases use a tagged version of the container (e.g. scilifelab/ngi-rnaseq:1.5) whereas dev versions use the most up to date version of the container (see issue #209 on github for details).
+  */
+  container = { "scilifelab/ngi-rnaseq" }
```

Comment:
This is now `params.container` instead of `process.container`, so will break I think.